scripts/SpaceCart.js

```diff
@@ -0,0 +1,74 @@
+import { getTransientState, getFacilities, getMinerals } from "./database"
+const transientState = getTransientState()
+const facilities = getFacilities()
+const minerals = getMinerals()
+
+const cartFacility = () => {
+    if (transientState.selectedFacility) {
+        for (const facility of facilities) {
+            if (facility.id === transientState.selectedFacility) {
+                return facility
+            }
+        }
+    }
+}
+let foundFacility = cartFacility()
+
+const cartMineral = () => {
+    if (transientState.selectedMineral) {
+        for (const mineral of minerals) {
+            if (transientState.selectedMineral === mineral.id) {
+                return mineral
+            }
+        }
+    }
+}
+let foundMineral = cartMineral()
+
+export const SpaceCart = () => {
+    return `<div class="cart--${foundMineral.id}>1 ton of ${foundMineral.name} from ${foundFacility.name}</div>`
```

Comment:
foundmineral.id probably just needs to be "foundMineral" - it should be only an interger based on cartMineral()